webui/react/src/pages/ExperimentDetails.tsx

```diff
@@ -61,7 +62,26 @@ const ExperimentDetails: React.FC = () => {
     valHistory,
   ]);
 
+  const fetchFirstTrialId = useCallback(async () => {
+    try {
+      const expTrials = await getExpTrials({ id }, { signal: canceler.signal });
+      if (expTrials.trials[0]) {
+        setFirstTrialId(expTrials.trials[0].id);
+      }
+    } catch (e) {
+      handleError({
+        error: e,
+        level: ErrorLevel.Error,
+        message: e.message,
+        publicMessage: 'Failed to fetch experiment trials.',
+        silent: true,
+        type: ErrorType.Server,
+      });
+    }
+  }, [ canceler, id ]);
+
   const { startPolling, stopPolling } = usePolling(fetchExperimentDetails);
+  const { stopPolling: stopPollingFirstTrialId } = usePolling(fetchFirstTrialId);
```

Comment:
Is this supposed to be a comma after stopPolling, or is stopPollingFirstTrialId a type?

Reply:
that's to rename `stopPolling` since it's already used from the line before :)

in this case this get exposed aliased as `stopPollingFirstTrialId`

Reply:
Ah, makes sense!